daemons/dss-index/app.py

```diff
@@ -10,20 +10,24 @@
 
 import dss
 from dss.events.handlers.index import process_new_s3_indexable_object, process_new_gs_indexable_object
+from dss.events.handlers.index import process_s3_removed_object
 
 app = domovoi.Domovoi()
 
 dss.Config.set_config(dss.DeploymentStage.NORMAL)
 
 s3_bucket = dss.Config.get_s3_bucket()
 
-@app.s3_event_handler(bucket=s3_bucket, events=["s3:ObjectCreated:*"])
+@app.s3_event_handler(bucket=s3_bucket, events=["s3:ObjectCreated:*", "s3:ObjectRemoved:Delete"])
 def dispatch_s3_indexer_event(event, context) -> None:
     app.log.setLevel(logging.DEBUG)
     if event.get("Event") == "s3:TestEvent":
         app.log.info("DSS index daemon received S3 test event")
     else:
```

Comment:
Why not do elif "ObjectRemoved" in event["Records"][0]["eventName"]:?

Reply:
No conditional in original code. Only two paths: ObjectRemoved checked explicitly, assume ObjectCreated otherwise.